frontend/src/pages/modelRegistry/screens/RegisteredModelsArchive/RegisteredModelsArchiveListView.tsx

```diff
@@ -68,7 +69,7 @@ const RegisteredModelsArchiveListView: React.FC<RegisteredModelsArchiveListViewP
                   }))}
                   value={searchType}
                   onChange={(newSearchType) => {
-                    setSearchType(newSearchType as SearchType);
+                    setSearchType(asEnumMember(newSearchType, SearchType)!);
```

Comment:
You need to guard against `null` before assigning. There's other code areas with the same issue.